examples/pybullet/gym/pybullet_envs/deep_mimic/env/humanoid_stable_pd.py

```diff
@@ -135,7 +137,10 @@ def __init__(self, pybullet_client, mocap_data, timeStep, useFixedBase=True):
     self._jointDofCounts = [4, 4, 4, 1, 4, 4, 1, 4, 1, 4, 4, 1]
 
     #only those body parts/links are allowed to touch the ground, otherwise the episode terminates
-    self._allowed_body_parts = [5, 11]
+    fall_contact_bodies = []
```

Comment:
I'd like to be backward compatible, so that if you don't pass in the argument it will use the [5,11] default. With the new change, things break unless you pass this flag.